index.html

```diff
@@ -7,33 +7,63 @@
     <link href="style.css" rel="stylesheet">
     <title>Awesome Books</title>
+    <script src="https://cdn.jsdelivr.net/npm/luxon@2.0.2/build/global/luxon.min.js"></script>
+    
+
 </head>
 
 <body>
-    <main class="main-content d-flex">
-        <section id="List-section" class="List-section d-flex"> 
-            <h1>All awesome books</h1>
-            <table id="bookList" class="book-table">
+    <header>    
+      <nav class="nav width-95-percent margin-auto"> 
+          <h4 class="margin-top-bottom-2px">Awesome Books</h4>     
+          <ul class="links flex-direction-row margin-bottom-0 margin-top-0">
+              <li class="nav-link border-right-1-px" onclick="browsePages(0)">List</li>
+              <li class="nav-link border-right-1-px" onclick="browsePages(1)">Add New</li>
+              <li class="nav-link" onclick="browsePages(2)">Contact</li>
+          </ul>
+      </nav>
+      <div>
+          <p id="time-display" class="float-right"></p>
+      </div>
+    </header>
+    <main class="main-content width-100-percent">
+        <section id="List-section" class="List-section d-flex margin-auto"> 
+            <h1 class="text-align-center">Awesome Books</h1>
+            <table id="bookList" class="book-table width-100-percent">
 
             </table>
             <hr class="List-section-hr">
         </section>
 
-        <section id="Add-section" class="Add-section d-flex"> 
-            <h3>Add books</h3>
+        <section id="Add-section" class="Add-section d-flex margin-auto"> 
+            <h3 class="text-align-center">Add books</h3>
 
-            <fieldset>
+            <fieldset class="margin-auto">
                 <form id="form" class="form d-flex">
-                    <input class="form-item" type="text" required placeholder="Title" id="book_title">
+                    <input class="form-item margin-auto" type="text" required placeholder="Title" id="book_title">
                     <br>
                     <input class="form-item" type="text" required placeholder="Author" id="book_author">
                     <br>
                     <input class="form-item addButton" type="submit" value="Add" id="addButton">
                 </form>
             </fieldset>
         </section>
+        <section class="contact-section margin-auto">
+            <h1 class="text-align-center">Contact Information</h1>
+            <p>Do you have any question, or you just want to say "Hello"?</p>
+            <p>You can reach out to us!</p>
+            <ul>
+                <li>Our e-mail: mail@mail.com</li>
+                <li>Our phone number: 00 123 45678</li>
+                <li>OUr address: Streethome 22, 4530 City, Country</li>
+            </ul>
+        </section>
     </main>
+    <footer class="width-95-percent margin-auto">
+       <p>&copy; Awesome Book Library 2021. All rights reserved...</p> 
+    </footer>
     <script src="app.js"></script>
+    <script src="app2.js"></script>
 </body>
 
 </html>
```

Comment:
- [ ] When you open the application this is how the page looks like
![land](https://user-images.githubusercontent.com/44745855/133666364-5fa76818-9d07-4e1a-8065-a0a128d9a7c7.png)

I suggest that when you open a page you display a list of books page even if it can be empty when you add books in the future they will appear on this page, you don't have to display form and contact because they have they specific page.